assets/terraform-modules/aws/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -100,6 +100,19 @@ systemd:
         ExecStop=/etc/kubernetes/delete-node
         [Install]
         WantedBy=multi-user.target
+    - name: "create-iscsi-initiatorname-file.service"
+      enable: true
+      contents: |
+        [Unit]
+        Description=Create ISCSI initiator name file
+        Before=iscsid.service
+        [Service]
+        Type=oneshot
+        RemainAfterExit=true
+        ExecStart=/opt/create-iscsi-initiatorname
```

Comment:
Will it run on every boot? I guess that may have some side effects? Why can't we specify a drop-in for `iscsid` service to run this before start?

Can we have a command inline here, rather than a script file? For one linters, it's less redirection.

Reply:
We can't modify/create "service" config when the service is already on the verge of starting.

Like if I create a dropin for `iscsid` and add the oneline script in `ExecStartPre` then it does not load the new config. We need someone else doing it.

Reply:
Why it does not load the new config? Actually, it might also be what is happening with kubelet at the moment, which might be causing #1191 :thinking:

Reply:
I'm still interested in resolving that.